change_column_type_to_lowcardinality/.tinyenv

```diff
@@ -1,2 +1,2 @@
 TB_VERSION_WARNING=0
-VERSION=0.0.1
```

Comment:
I feel we don't need to update the version, do we?

Reply:
Yes, because we are changing the endpoints and even `tb deploy` will validate that the version does not exist yet.